### PR TITLE
Make imports in stories look like import from installed lib

### PR DESCRIPTION
**Pull request by Ermakoy**
solves #139

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-        loaders: [require.resolve('@storybook/addon-storysource/loader')],
-        enforce: 'pre',
-    });
+module.exports = ({ config }) => {
+  config.module.rules.push({
+    test: /\.stories\.js?$/,
+    loaders: [require.resolve('@storybook/addon-storysource/loader')],
+    enforce: 'pre'
+  });
+  console.log(path.resolve(__dirname, '../src/components/index'));
```
Comment (luizbaldi) [inline .storybook/webpack.config.js:9]: Could you remove this console.log? 😬

**Comment by luizbaldi** [comment]
@Ermakoy Thanks for contributing to the project, great work! 😄

**Comment by arturbien** [comment]
@Ermakoy heyyy! Thanks for helping ♥️ master branch was not active lately and the heavy development is currently happening on the ‚beta’ branch (we’re preparing for v3 release). Can you rebase your work on top of beta branch and then submit PR to beta? Some changes may be needed because quite a lot changed there

File: .storybook/webpack.config.js (hunk 1 above)
Comment (Ermakoy) [reply .storybook/webpack.config.js:9]: oh, sorry

**Comment by Ermakoy** [comment]
Close in favor of #144

**Pull request closed without merging**